adodbapi/quick_reference.md

```diff
@@ -46,12 +46,11 @@ combinations. See
 and for more explanation:
```

Comment:
This is a broken link and should be removed.

Reply:
Done